src/state/farms/elastic/hooks.ts

```diff
@@ -762,6 +763,10 @@ export function useJoinedPositions() {
               } else {
                 rewardPendings[pid][i] = rewardPendings[pid][i].add(amount)
               }
+
+              // TODO:
```

Comment:
todo?

Reply:
hardcode for now, todo for future. maybe never check again